### DISCUSSION TITLE
Rename fusion transcript section

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#13] [HGVS nomenclature proposals]
Currently in the HGVS nomenclature there is a fusion transcript section in the [RNA splicing documentation](https://varnomen.hgvs.org/recommendations/RNA/variant/splicing/). However what this section describes is generalizable to contexts outside of what may be considered a "gene fusion" (see #10). To address this, it is proposed to change the name of this section to concatenated transcript segments.

_Concatenated transcript segments_ was initially proposed, but some feedback received indicated this may be unfamiliar vocabulary to much of the HGVS user community. _Adjoined transcript segments_ is a possible alternative, though it is perhaps less precise than _concatenated transcript segments_ when discussing >2 components (e.g. with a linker sequence) in a description.

**Current status:**
Addressed in HGVSnomenclature/VarNomen#34

**Comment by RosHastir** [comment]
'Adjoined transcript segment' is a more accurate term for a fusion transcript than 'concatenated' or 'join': 
Adjoin definition : be next to and joined with i.e. the adjoining room
Ros Hastings

**Comment by marinadistefano** [comment]
I think I also like the term "adjoined transcript segment" better than "concatenated transcript segment".
